macOS/LocalPackages/NewTabPage/Sources/NewTabPage/Favorites/NewTabPageFavoritesModel.swift

```diff
@@ -50,6 +50,7 @@ final class UserDefaultsNewTabPageFavoritesSettingsPersistor: NewTabPageFavorite
     }
 }
 
+public let NewTabPageContextMenuTitle = "NewTabPageContextMenu"
```

Comment:
We have more context menus on the NTP. There’s also sections visibility menu (on the NTP background) and a custom image menu (on an image thumbnail in the customization panel).

If you want to handle only the favorites menu, let’s rename it as `NewTabPageFavoritesContextMenuTitle`, and also please add swiftlint rule exclusion here.

Reply:
looks like we don't need that constant at all, I've dropped it